@xen-orchestra/lite/src/components/pool/network/PoolNetworksPifStatus.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <UiInfo class="text-ellipsis" :accent="statusProps.accent">
+    <span class="text-ellipsis">{{ statusProps.text }}</span>
+  </UiInfo>
+</template>
+
+<script setup lang="ts">
+import UiInfo from '@core/components/ui/info/UiInfo.vue'
+import { computed } from 'vue'
+
+const { pif } = defineProps<{
+  pif: { status: 'connected' | 'disconnected' | 'partially_connected' }
```

Comment:
By convention, I think it would be better to use the kebab-case here:
```suggestion
  pif: { status: 'connected' | 'disconnected' | 'partially-connected' }
```
Remember to update the rest of the code and the usages accordingly.